packages/iTwinUI-react/src/core/ComboBox/ComboBoxMultipleContainer.tsx

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import SelectTagContainer from '../Select/SelectTagContainer';
+
+type ComboBoxMultipleContainerProps = {
+  selectedItems?: JSX.Element[];
+};
+
+export const ComboBoxMultipleContainer = React.forwardRef(
+  (
+    { selectedItems = [] }: ComboBoxMultipleContainerProps,
+    ref: React.RefObject<HTMLDivElement>,
+  ) => {
+    return <SelectTagContainer ref={ref} tags={selectedItems} />;
+  },
+);
```

Comment:
Readability and flexibility of this component can be improved.
```suggestion
type ComboBoxMultipleContainerProps = {
  selectedItems?: JSX.Element[];
} & Omit<React.ComponentPropsWithoutRef<'div'>, 'children'>;

export const ComboBoxMultipleContainer = React.forwardRef(
  (props: ComboBoxMultipleContainerProps, ref: React.Ref<HTMLDivElement>) => {
    const { selectedItems = [], ...rest } = props;
    return <SelectTagContainer ref={ref} tags={selectedItems} {...rest} />;
  },
);
```